lego/apps/lending/notifications.py

```diff
@@ -0,0 +1,40 @@
+from lego.apps.lending.models import LendingInstance
+from lego.apps.notifications.constants import LENDING_INSTANCE_CREATION
+from lego.apps.notifications.notification import Notification
+from lego.apps.users.models import User
+
+
+class LendingInstanceNotification(Notification):
```

Comment:
Could we make a more descriptive name for this. I assume it is sent to the responsible group when a new lending request is created?
I think this will be confusing if we add more notifications later, f.ex. status changes like when the lending instance is approved/denied.